examples/pytorch/summarization/run_summarization.py

```diff
@@ -692,7 +692,16 @@ def compute_metrics(eval_preds):
     results = {}
     if training_args.do_eval:
         logger.info("*** Evaluate ***")
-        metrics = trainer.evaluate(metric_key_prefix="eval")
+        if isinstance(eval_dataset, dict):
+            metrics = {}
+            for eval_ds_name, eval_ds in eval_dataset.items():
+                dataset_metrics = trainer.evaluate(
+                    eval_dataset=eval_ds,
+                    metric_key_prefix=f"eval_{eval_ds_name}",
+                )
```

Comment:
This should fit in one line here.

Reply:
Thanks! Just reformatted. What is the max line length Huggingface prefers?

Reply:
CircleCI throws this error message. I don't see how my commit is relevant to this?

```
FAILED examples/pytorch/test_pytorch_examples.py::ExamplesTests::test_generation - OSError: There was a specific connection error when trying to load sshleifer/tiny-gpt2:
421 Client Error: Misdirected Request for url: https://huggingface.co/sshleifer/tiny-gpt2/resolve/main/model.safetensors.index.json
FAILED examples/pytorch/test_pytorch_examples.py::ExamplesTests::test_run_audio_classification - OSError: There was a specific connection error when trying to load hf-internal-testing/tiny-random-wav2vec2:
421 Client Error: Misdirected Request for url: https://huggingface.co/hf-internal-testing/tiny-random-wav2vec2/resolve/main/preprocessor_config.json
FAILED examples/pytorch/test_pytorch_examples.py::ExamplesTests::test_run_clm - OSError: There was a specific connection error when trying to load distilgpt2:
421 Client Error: Misdirected Request for url: https://huggingface.co/distilgpt2/resolve/main/config.json
FAILED examples/pytorch/test_pytorch_examples.py::ExamplesTests::test_run_clm_config_overrides - OSError: There was a specific connection error when trying to load gpt2:
421 Client Error: Misdirected Request for url: https://huggingface.co/gpt2/resolve/main/config.json
FAILED examples/pytorch/test_pytorch_examples.py::ExamplesTests::test_run_image_classification - OSError: There was a specific connection error when trying to load google/vit-base-patch16-224-in21k:
421 Client Error: Misdirected Request for url: https://huggingface.co/google/vit-base-patch16-224-in21k/resolve/main/config.json
FAILED examples/pytorch/test_pytorch_examples.py::ExamplesTests::test_run_mlm - OSError: There was a specific connection error when trying to load distilroberta-base:
421 Client Error: Misdirected Request for url: https://huggingface.co/distilroberta-base/resolve/main/model.safetensors
FAILED examples/pytorch/test_pytorch_examples.py::ExamplesTests::test_run_ner - OSError: There was a specific connection error when trying to load bert-base-uncased:
421 Client Error: Misdirected Request for url: https://huggingface.co/bert-base-uncased/resolve/main/config.json
FAILED examples/pytorch/test_pytorch_examples.py::ExamplesTests::test_run_semantic_segmentation - huggingface_hub.utils._errors.HfHubHTTPError: 421 Client Error: Misdirected Request for url: https://huggingface.co/datasets/huggingface/semantic-segmentation-test-sample/resolve/main/id2label.json
FAILED examples/pytorch/test_pytorch_examples.py::ExamplesTests::test_run_speech_recognition_ctc -   File "/home/circleci/.cache/huggingface/modules/evaluate_modules/metrics/evaluate-metric--wer/1f0994d772b2a524d0447b9618e48c30fde5dc5b736ad5bd0cc76e6ccb600d26/wer.py", line 1
    <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
    ^
SyntaxError: invalid syntax
```

Reply:
It was a flaky error, re-running the tests show everything is fine.